virtcontainers/store/filesystem_backend.go

```diff
@@ -152,11 +152,13 @@ func (f *filesystem) initialize() error {
 	// The root directory, a lock file and a raw files directory.
 
 	// Root directory
+	f.logger().Debugf("Creating root directory if it already does not exist %s", f.path)
```

Comment:
thx. I think we should take on board @devimc suggestion of using `WithField`.
I'd also rephrase these to drop the 'if it already....' bit, as unless we do the check (stat) and print a status, then I think it will just be confusing noise. So, just `Creating root directory` will be enough (along with the WithField(path)) I think.